### PR TITLE
Added Python to highlite module.

### PR DESCRIPTION
**Pull request by xbello**
* Copy-pasted the Nim Tokenizer to remove the non-existent multiline comment
operator, and added a 'keywords' parameter (like in cLikeNextToken).

* Added the keywords for Python

### CODE REVIEW

**Comment by Araq** [comment]
> Copy-pasted the Nim Tokenizer to remove the non-existent multiline comment
operator, and added a 'keywords' parameter (like in cLikeNextToken).

Instead patch the Nim Tokenizer and disable the multiline comment for ``g.lang == langPython`` (if ``g.lang`` doesn't yet exist, introduce it).

File: lib/packages/docutils/highlite.nim [hunk 1]
```diff
@@ -886,6 +1042,16 @@ proc yamlNextToken(g: var GeneralTokenizer) =
   g.length = pos - g.pos
   g.pos = pos
 
+proc pythonNextToken(g: var GeneralTokenizer) =
+  const
+    keywords: array[0..34, string] = [
+      "False", "None", "True", "and", "as", "assert", "async", "await",
```
Comment (Araq) [inline lib/packages/docutils/highlite.nim:1048]: Iirc, False, None and True are not keywords.

**Review by xbello — COMMENTED** [review]
Had to made a couple of additional changes to pass the tests (tests/stdlib/trstgen.nim).

**Comment by Araq** [comment]
Excellent work!

**Pull request merged**